test/unit/generator/content/schedule/part2/part2.test.js

```diff
@@ -0,0 +1,24 @@
+const part2 = require('../../../../../../app/generator/content/schedule/part2')
+const mockSchedule = require('../../../../../mocks/mock-schedule')
+
+describe('schedule part 2', () => {
```

Comment:
these tests are only for top-ups, need sim. for reductions.

I'd recommend using another set of `describe` to split them up effectively, i.e.

describe('schedule part 2')
describe('when schedule is a top-up')
all the tests you have now
describe('when schedule is a reduction')
sim. tests
describe('when schedule is 0')
new tests, uncovered scenario